src/test/integration/insert_binary_spec.js

```diff
@@ -1,44 +1,84 @@
 require('./integration_config.js');
-var fs = require('fs');
-var request = require('request');
+
 var channelName = utils.randomChannelName();
-var thisChannelResource = channelUrl + "/" + channelName;
-var testName = __filename;
-
-utils.runInTestChannel(testName, channelName, function () {
-
-    catUrl = 'http://www.lolcats.com/images/u/08/32/lolcatsdotcombkf8azsotkiwu8z2.jpg';
```

Comment:
We've been downloading this cat for years. Perhaps we should not rely on lolcats.com?